lib/preinstall.js

```diff
@@ -57,11 +57,6 @@ function isOpenZWaveInstalled() {
     sysconfdir: {
       findpattern: 'zwcfg.xsd',
       locations: ["/usr/etc", "/usr/local/etc/", "/etc/"]
-    },
-    patch1125: {
-      findpattern: 'CentralScene.h',
-      locations: ["/tmp", "/usr/include/", "/usr/local/include/"],
-      searchpattern: 'createSupportedKeyAttributesValues'
```

Comment:
Makes sense... PR1125 is merged in 1.6

Reply:
So you want to keep it?

Reply:
Nonono, thats not what I meant. I had this part removed before but then we had to go back to 1.4 because of incompatibilities. I should have removed it after targeting 1.6 for the second time. So all good.